pages/portfolio.html

```diff
@@ -5,8 +5,33 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Document</title>
+  <link href="../styles/style.css" rel="stylesheet">
 </head>
 <body>
-  
+  <section class="grid-container">
+    <div class="orange"><h1>My Project in Ada </h1></div>
+    <div class="yellow"><h2>Project Details</h2></div>
```

Comment:
Let `h1` and `h2` elements stand on their own. You can style them with classes directly.
```html
<h1 class="orange">My Project in Ada </h1>
```